### ISSUE TITLE
Github pages not found

### ISSUE THREAD

**Issue by georgek** [vivien/i3blocks#327]
The Github pages for this project is showing a 404.

**Comment by vivien** [comment]
You're correct. I've rewrote the README from scratch and I was thinking it's actually enough as a single place for the documentation. But I will convert the README and the manpage as HTML and put them in a `docs/` folder so that we get Github pages as well.
